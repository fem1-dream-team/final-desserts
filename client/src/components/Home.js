import React from 'react';
// import styled from "styled-components";
// import {NavLink} from "react-router-dom";
// import {LogoImg} from "../commons/Header/LogoImg";

export const Home = () => {
	return (
		<div>
            Home
		</div>
	)
};

// const Container = styled.div`
//     width: 180px;
//     margin: 0 auto;
// `
