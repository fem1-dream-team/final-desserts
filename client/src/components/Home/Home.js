import React from 'react';
import {SimpleSlider} from '../SimpleSlider';

export const Home = () => {
	return (
		<div>
			<SimpleSlider/>
		</div>
	)
};
