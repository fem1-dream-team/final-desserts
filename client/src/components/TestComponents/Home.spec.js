import React from "react";
import { Home } from "../Home";
import renderer from 'react-test-renderer'

describe("Home component", () => {

	it("Home component render", () => {
		const wrapper = renderer.create(<Home />).toJSON();
		expect(wrapper).toMatchSnapshot();

	});
});