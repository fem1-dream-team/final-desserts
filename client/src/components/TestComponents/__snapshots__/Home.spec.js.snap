// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component Home component render 1`] = `
<div>
  Home
</div>
`;
